NewEvaluation.css

```diff
@@ -190,9 +204,22 @@ input[type=submit]{
 	float: right;
 	box-shadow: 5px 5px 5px #888888;
 }
-.spacing{
-	padding-top: 25vh;
-}
-.littleSpacing{
-	padding-top: 10vh;
+
+
+
+
+
+/*-----------------   Footer -----------------*/
+
+
+.BottomFooter{
+	bottom:0;
+    position:fixed;
+    z-index:150;
+    _position:absolute;
+    _top:expression(eval(document.documentElement.scrollTop+
```

Comment:
Nope in the end I started playing with this without the expression. Managed to make it work the same  way it worked with the expression. Don't know why it acted and worked differently before